src/main/scala/com/amazon/deequ/repository/AnalysisResultSerde.scala

```diff
@@ -256,6 +256,12 @@ private[deequ] object AnalyzerSerializer
         result.addProperty(COLUMN_FIELD, sum.column)
         result.addProperty(WHERE_FIELD, sum.where.orNull)
 
+      case ratioOfSums: RatioOfSums =>
+        result.addProperty(ANALYZER_NAME_FIELD, "RatioOfSums")
+        result.addProperty("numerator", ratioOfSums.numerator)
```

Comment:
@scott-gunn, pretty much all of the serializers have either COLUMN_FIELD or COLUMNS_FIELD. Maybe we should use COLUMN_FIELDS instead of numerator, denominator?